terraform/environments/digital-prison-reporting/main.tf

```diff
@@ -1262,56 +1262,6 @@ module "dms_nomis_ingestor" {
   )
 }
 
-module "dms_fake_data_ingestor" {
```

Comment:
I'm removing this since it caused issues in my pipeline and it is no longer needed